app/uk/gov/hmrc/agentservicesaccount/views/pages/help.scala.html

```diff
@@ -48,19 +48,19 @@ <h2 class="govuk-heading-m">@Messages("help.authorised.h2")</h2>
     <h2 class="govuk-heading-m">@Messages("help.mtd.h2")</h2>
     <div class="govuk-grid-column-one-third govuk-!-padding-left-0 govuk-!-padding-bottom-1">
         <ul class="govuk-list">
-            <li class="govuk-!-margin-bottom-4"><a class="https://www.gov.uk/guidance/making-tax-digital-for-vat-as-an-agent-step-by-step" target="_blank" href="">@Messages("help.mtd.link1")</a></li>
-            <li class="govuk-!-margin-bottom-4"><a class="https://www.gov.uk/government/publications/vat-notice-70022-making-tax-digital-for-vat/vat-notice-70022-making-tax-digital-for-vat" target="_blank" href="">@Messages("help.mtd.link2")</a></li>
+            <li class="govuk-!-margin-bottom-4"><a class="govuk-link govuk-link--no-visited-state" target="_blank" href="https://www.gov.uk/guidance/making-tax-digital-for-vat-as-an-agent-step-by-step">@Messages("help.mtd.link1")</a></li>
```

Comment:
for target="_blank" you also need the attribute rel="noopener"

Reply:
Thanks!